interactive-clap-derive/src/tests/snapshots/interactive_clap_derive__tests__test_simple_struct__vec_multiple_opt.snap

```diff
@@ -11,6 +11,27 @@ pub struct CliArgs {
 impl interactive_clap::ToCli for Args {
     type CliVariant = CliArgs;
 }
+impl Args {
+    pub fn try_parse() -> Result<CliArgs, clap::Error> {
+        <CliArgs as clap::Parser>::try_parse()
+    }
+    pub fn parse() -> CliArgs {
+        <CliArgs as clap::Parser>::parse()
+    }
+    pub fn try_parse_from<I, T>(itr: I) -> Result<CliArgs, clap::Error>
+    where
+        I: ::std::iter::IntoIterator<Item = T>,
+        T: ::std::convert::Into<::std::ffi::OsString> + ::std::clone::Clone,
+    {
+        <CliArgs as clap::Parser>::try_parse_from(itr)
+    }
+}
+impl From<Args> for CliArgs {
+    fn from(args: Args) -> Self {
+        Self { env: args.env.into() }
+    }
+}
+impl Args {}
```

Comment:
Seems like empty block somehow generated

Reply:
I don't mean it's bad, just to check for you if it's intended

Reply:
It's intended when no input_arg functions were generated. Previously these were in a single block with clap parser adapter, which had slightly different purposes

Reply:
i'll try to remember to try include comments into generated output itself in another pr, 
and see if it breaks anything. Shouldn't break anything at all, but should probably be checked first
```rust
    quote! {
        // derived implementation block for `#name` struct, which contains functions `input_#field_ident` per each field,
        // which prompt for value of each field via [inquire::CustomType](https://docs.rs/inquire/0.6.2/inquire/struct.CustomType.html)
        impl #name {
            #(#vec_fn_input_arg)*
        }
    }
```